guides/06_other_tutorials/setting_up_a_demo_for_maximum_performance.md

```diff
@@ -1,5 +1,8 @@
 # Setting Up a Demo for Maximum Performance
 
+Tags: QUEUE, PERFORMANCE
```

Comment:
Just a note that this won't get the guide to show up under `queue` or `Interface`. The right way to get it to show up with these is to add it to the `Guides: ` section of the docstring for Interface.

Reply:
Yeah, that's fine, this was just for the tags